INFO     run_all_case.py:test_RechargeCase.py:46 当前用例名称：输入负数
INFO     run_all_case.py:test_RechargeCase.py:47 当前测试用例请求参数：{'asset': -1}
INFO     run_all_case.py:test_RechargeCase.py:48 当前用例预期结果：{'code': 'BadRequest', 'msg': '充值金额最低1元'}
INFO     run_all_case.py:test_RechargeCase.py:49 当前用例实际结果：{'code': 'Success', 'data': {'code_url': '', 'recharge_uuid': '668675177436161559', 'asset': -1}, 'msg': ''}

ERROR    run_all_case.py:Assert.py:110 Traceback (most recent call last):
  File "C:\Users\张铁瀛\PycharmProjects\API_AUTO\common\Assert.py", line 104, in define_assert
    assert res['code'] == respData['code']
AssertionError: assert 'Success' == 'BadRequest'
  - BadRequest
  + Success

ERROR    run_all_case.py:test_RechargeCase.py:53 用例执行失败assert 'Success' == 'BadRequest'
  - BadRequest
  + Success